templates/base.html

```diff
@@ -61,9 +61,6 @@
         <li class="nav-item">
           <a href="/meeting/projects" class="nav-link">Projects</a>
         </li>
-        <li class="nav-item">
-          <a href="/meeting/availability" class="nav-link">Availability</a>
-        </li>
```

Comment:
Is Justin ready for this one?

Reply:
Not sure where this came into play, but it was not included in the new push.